tap_github/repository_streams.py

```diff
@@ -153,6 +153,12 @@ def get_child_context(self, record: dict, context: Optional[dict]) -> dict:
 
 
 class ReadmeStream(GitHubStream):
+    """
+    A stream dedicated to fetching the object version of a README.md.
+
+    Inclduding its content, base64 encoded. For html, see ReadmeHtmlStream.
```

Comment:
maybe worth clarifying that this content is github-flavoured-markdown as opposed to the html below?